telemetry/csharp/ToolkitTelemetryGenerator.Tests/expectedCode.txt

```diff
@@ -0,0 +1,436 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
```

Comment:
is this typical - I'd expect the name of the tool to be here - so that someone can trace back to 'how we got here'

Reply:
This is a stock comment that we don't appear to be able to modify. I added a comment as high up in the file as I could (on line 15)